src/pages/RGBPP/TransactionList/Chart/index.tsx

```diff
@@ -0,0 +1,200 @@
+import BigNumber from 'bignumber.js'
+import { useTranslation } from 'react-i18next'
+import { SupportedLng, useCurrentLanguage } from '../../../../utils/i18n'
+import styles from './styles.module.scss'
+import {
+  DATA_ZOOM_CONFIG,
+  assertIsArray,
+  assertSerialsDataIsStringArrayOf3,
+  assertSerialsItem,
+  parseNumericAbbr,
+} from '../../../../utils/chart'
+import { parseSimpleDate } from '../../../../utils/date'
+import { isMainnet } from '../../../../utils/chain'
+import { tooltipColor, tooltipWidth, SeriesItem, SmartChartPage } from '../../../StatisticsChart/common'
+import { ChartItem, explorerService } from '../../../../services/ExplorerService'
+import { ChartColorConfig } from '../../../../constants/common'
+import { useIsMobile, useIsXXLBreakPoint } from '../../../../hooks'
+
+const widthSpan = (value: string, language: SupportedLng) => tooltipWidth(value, language === 'en' ? 168 : 110)
+
+const useTooltip = () => {
+  const { t } = useTranslation()
+  const currentLanguage = useCurrentLanguage()
```

Comment:
`currentLanguage` can be loaded from `const { t, i18n: { language }} = useTranslation()`

Reply:
> `currentLanguage` can be loaded from `const { t, i18n: { language }} = useTranslation()`

`useCurrentLanguage` will return the languages supported, now they are Chinese and English, and enumerated in a specific type. But the `i18n` in `useTranslation` will return any kinds of languages and the type is `string`

Reply:
I got the usage of `withSpan`, it's unnecessary, you can use table in the tooltip as did at https://github.com/nervosnetwork/ckb-explorer-frontend/blob/develop/src/pages/StatisticsChart/activities/ContractResourceDistributed.tsx#L45

It works as 
<img width="399" alt="image" src="https://github.com/Magickbase/ckb-explorer-frontend/assets/7271329/2d2b234e-d7ed-4364-8fd6-6ce3fd79f0b3">

Reply:
Any update on this conversation

Reply:
> > `currentLanguage` can be loaded from `const { t, i18n: { language }} = useTranslation()`
> 
> `useCurrentLanguage` will return the languages supported, now they are Chinese and English, and enumerated in a specific type. But the `i18n` in `useTranslation` will return any kinds of languages and the type is `string`

It is hidden by GitHub, so I did't notice, I will have a look.

Reply:
done